src/test/resources/features/foi/Acceptance.feature

```diff
@@ -0,0 +1,25 @@
+@FOIAcceptance @FOI @FOIWorkflow
+Feature: Acceptance
+
+  Background:
+    Given I am logged into "CS" as user "FOI_USER"
+    When I create a "FOI" case and move it to the "Acceptance" stage
+    And I load and claim the current case
+
+  #HOCS-2586, HOCS-2723
+  Scenario: User is able to complete the Acceptance stage
+    And I select that the case "Does" belong to this Directorate
+    And I click the "Continue" button
```

Comment:
can we move the button clicks like this into the steps above them?

Reply:
Resolved

Reply:
In this instance it wouldn't work, the scenario directly after this has the step 'And I enter a rejection reason at the Acceptance stage', this must be completed before the user clicks the continue button.